app/code/Magento/Wishlist/Test/Mftf/Test/StorefrontShareWishlistWithMoreThanMaximumAllowedEmailsQtyTest.xml

```diff
@@ -0,0 +1,56 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+ /**
+  * Copyright © Magento, Inc. All rights reserved.
+  * See COPYING.txt for license details.
+  */
+-->
+
+<tests xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+       xsi:noNamespaceSchemaLocation="urn:magento:mftf:Test/etc/testSchema.xsd">
+    <test name="StorefrontShareWishlistWithMoreThanMaximumAllowedEmailsQtyTest">
+        <annotations>
+            <features value="Wishlist"/>
+            <stories value="Sharing wishlist with more than Maximum Allowed Emails qty"/>
+            <title value="Sharing wishlist with more than Maximum Allowed Emails qty"/>
+            <description value="Customer should not have a possibility share wishlist with more than maximum allowed emails qty"/>
+            <group value="wishlist"/>
+            <group value="configuration"/>
+        </annotations>
+        <before>
+            <magentoCLI command="config:set wishlist/email/number_limit 1" stepKey="changeEmailsQtyLimit"/>
+            <magentoCLI command="cache:clean" stepKey="cleanCache"/>
+            <createData entity="SimpleSubCategory" stepKey="createCategory"/>
+            <createData entity="SimpleProduct" stepKey="createProduct">
+                <requiredEntity createDataKey="createCategory"/>
+            </createData>
```

Comment:
After creating product and category, you should run `<magentoCron` for index group.

Reply:
Added Cron Magento CLI command on 27 line
Please check.
Thanks!